addons/escoria-core/game/core-scripts/save_data/esc_save_manager.gd

```diff
@@ -24,6 +24,8 @@ const SETTINGS_TEMPLATE: String = "settings.tres"
 
 # Constructor of ESCSaveManager object.
 func _init():
+	# We leave the calls to ProjectSettings as-is since this constructor can be
```

Comment:
Maybe we can move this to _ready()?

Reply:
I would think the constructor is the right place for this given that the script doesn't have any children, and isn't _ready() more for GUI-related tasks? (In fact, I should have the command initializations in _init(), not _ready(), so I'm moving them there, too.) Can discuss, though.